lib/mongodb.js

```diff
@@ -0,0 +1,79 @@
+import { MongoClient } from 'mongodb'
+
+const uri = process.env.MONGODB_URI
+const options = {
+  useUnifiedTopology: true,
+  useNewUrlParser: true,
+}
+
+let client
+let clientPromise
+
+if (!process.env.MONGODB_URI) {
+  throw new Error('Please add your Mongo URI to .env.local')
+}
+
+if (process.env.NODE_ENV === 'development') {
+  // In development mode, use a global variable so that the value
+  // is preserved across module reloads caused by HMR (Hot Module Replacement).
+  if (!global._mongoClientPromise) {
+    client = new MongoClient(uri, options)
+    global._mongoClientPromise = client.connect()
+  }
+  clientPromise = global._mongoClientPromise
+} else {
+  // In production mode, it's best to not use a global variable.
+  client = new MongoClient(uri, options)
+  clientPromise = client.connect()
+}
+
+// Export a module-scoped MongoClient promise. By doing this in a
+// separate module, the client can be shared across functions.
+export default clientPromise
+
+const MONGODB_URI = process.env.MONGODB_URI;
+const MONGODB_DB = process.env.DB_NAME;
+
+// check the MongoDB URI
+if (!MONGODB_URI) {
+    throw new Error('Define the MONGODB_URI environmental variable');
+}
+
+// check the MongoDB DB
+if (!MONGODB_DB) {
+    throw new Error('Define the MONGODB_DB environmental variable');
+}
+
+let cachedClient = null;
+let cachedDb = null;
+
+export async function connectToDatabase() {
```

Comment:
Use arrow function syntax.
Example:
const foo = async () => {}

Secondly mayby we should name this getData since that's what it does, and we might have different ones in the future, like deleteData, addData etc

Reply:
Turned into arrow function.

This doesn't fetch the data, it just connects us to the database(I think). We are seeing the data in the /api/recipes only as a JSON file. We would have to create another fetch function to get the data and then display them on the website.

This just makes it easy to refer to the /api/recipes on the getData function we would create in the future.
Correct me if I'm wrong.